docs/diagrams/testing-strategy.puml

```diff
@@ -0,0 +1,61 @@
+@startuml testing-strategy
+
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+skinparam linetype polyline
+
+!global $WHITE = "#FFFFF"
+!global $UNIT_TESTS_COLOR = "#8B43D5"
+!global $SMOKE_TESTS_COLOR = "#1AB5B5"
+!global $SIMULATORS_COLOR = "#2F7070"
+!global $MOCK_SERVERS_COLOR = "#512A78"
+
+AddElementTag("unit_tests", $bgColor=$UNIT_TESTS_COLOR, $fontColor=$WHITE)
+AddElementTag("smoke_tests", $bgColor=$SMOKE_TESTS_COLOR, $fontColor=$WHITE)
+AddElementTag("simulators", $bgColor=$SIMULATORS_COLOR, $fontColor=$WHITE)
+AddElementTag("mock_servers", $bgColor=$MOCK_SERVERS_COLOR, $fontColor=$WHITE)
+
+title [Testing Strategy] HMPPS Integration API
+
+System_Ext(consumer, "Consumer Application", "An authenticated consumer of the Integration API.")
+
+Boundary(spring_boot_api, "HMPPS Integration REST API", "Spring Boot Kotlin API"){
+  Container(controller, "Controller", "Spring Boot API Controller", "")
+  Container(service, "Service", "Spring Boot Service Bean", "")
+  Container(gateway, "Gateway", "Spring Boot Component Bean", "")
+}
+
+Boundary(unit_test_boundary, "Unit Tests"){
```

Comment:
Is it worth including that these are in written within Kotlin as a tag underneath?